go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/hashicorp/vault-secrets-operator
 
-go 1.21
+go 1.22.0
```

Comment:
Odd that [apimachinery](https://github.com/kubernetes/apimachinery/blame/v0.30.0/go.mod) is forcing 1.22.0 here 🤷

Reply:
Yeah, same for [client-go](https://github.com/kubernetes/client-go/blame/master/go.mod)

Reply:
Things still look good either way.